src/controllers/itemDetails/index.js

```diff
@@ -1829,14 +1829,49 @@ function getVideosHtml(items) {
     });
 }
 
-function renderSpecials(page, item, user) {
-    ServerConnections.getApiClient(item.ServerId).getSpecialFeatures(user.Id, item.Id).then(function (specials) {
-        const specialsContent = page.querySelector('#specialsContent');
-        specialsContent.innerHTML = getVideosHtml(specials);
-        imageLoader.lazyChildren(specialsContent);
+function renderArtistTrackList(page, item, user) {
+    const request = {
+        SortBy: 'Name',
+        SortOrder: 'Ascending',
+        IncludeItemTypes: 'Audio',
+        Recursive: true,
+        ArtistIds: item.Id
+    };
+
+    ServerConnections.getApiClient(item.ServerId).getItems(user.Id, request).then(function (result) {
+        if (result.Items.length) {
+            page.querySelector('#specialsCollapsible').classList.remove('hide');
+            page.querySelector('#specialsCollapsible').classList.add('verticalSection-extrabottompadding');
+            page.querySelector('#specialsCollapsible>.sectionTitle').innerHTML = 'Songs';
+            const specialsContent = page.querySelector('#specialsContent');
+            specialsContent.classList.remove('itemsContainer');
+            specialsContent.innerHTML = listView.getListViewHtml({
+                items: result.Items,
+                smallIcon: true,
+                showIndexNumberLeft: false,
+                playFromHere: true,
+                action: 'play',
+                album: true
+            });
+            imageLoader.lazyChildren(specialsContent);
+        } else {
+            page.querySelector('#specialsCollapsible').classList.add('hide');
+        }
     });
 }
 
+function renderSpecials(page, item, user) {
+    if ( item.Type == 'MusicArtist' ) {
```

Comment:
Should probably be ===
```suggestion
    if ( item.Type === 'MusicArtist' ) {
```